Makefile

```diff
@@ -38,6 +38,7 @@ update-deps: mod-download
 	mkdir -p $(DEPSGOBIN)
 	chmod +x $(shell go list -f '{{ .Dir }}' -m k8s.io/code-generator)/generate-groups.sh
 	GOBIN=$(DEPSGOBIN) go install github.com/solo-io/protoc-gen-ext
+	GOBIN=$(DEPSGOBIN) go install github.com/sam-heilbron/protoc-gen-openapi
```

Comment:
For testing I just created this repo, based on https://github.com/istio/tools/tree/593a41c76c5c84a4cd51a4ab0c345630c5ed30ba/openapi/protoc-gen-openapi. I'm happy to create a solo owned repo to house this code. Our goal is that it is short lived and deleted when we support cue.

Reply:
Is this the repo removed from istio? https://github.com/istio/tools/tree/593a41c76c5c84a4cd51a4ab0c345630c5ed30ba/openapi/protoc-gen-openapi  What did they replace it with?

Reply:
They use `cue` https://github.com/istio/tools/tree/master/cmd/cue-gen